### DISCUSSION TITLE
PHP 8.3

### DISCUSSION THREAD

**Discussion by mambax7** [spatie/global-ray#41] [Q&A]
I tried to install Ray on PHP 8.3 RC 1, but got this error:
```
Xdebug requires Zend Engine API version 420220829.
The Zend Engine API version 420230831 which is installed, is newer.
Contact Derick Rethans at https://xdebug.org/docs/faq#api for a later version of Xdebug.
```
Now I can't run/test anything because I'm getting errors like this:
`Error: Call to undefined function ray() in file xxx.php line 48`

It runs fine on PHP 8.2.10, but how can I use Ray on PHP 8.3 RC1 ?

Or do I have to comment out all my ray() calls? 

I'm using Windows.

**Comment by stevebauman** [comment]
Hi @mambax7,

This doesn't appear to be a ray issue. It appears the XDebug extension is not up to date, as it's not compiled against the newer PHP version (8.3 RC1) that you have installed on your machine.

Have you tested commenting out all your ray calls to ensure global-ray is the problem in this?

**Comment by mambax7** [comment]
I tested now without the XDebug on both PHP 8.2.10 and PHP 8.3.0 RC1, and Ray works fine on PHP 8.2.10,  but not PHP 8.3.0 RC1, so it seems like it's an issue with Ray. 

When I comment out ray calls, then it works fine.
